x-pack/plugins/cases/public/components/all_cases/utility_bar.tsx

```diff
@@ -49,13 +66,63 @@ export const CasesTableUtilityBar: FunctionComponent<Props> = React.memo(
       onActionSuccess: onRefresh,
     });
 
+    const handleNotShowAgain = () => {
+      setLocalStorageWarning(true);
+    };
+
     /**
      * At least update or delete permissions needed to show bulk actions.
      * Granular permission check for each action is performed
      * in the useBulkActions hook.
      */
     const showBulkActions = (permissions.update || permissions.delete) && selectedCases.length > 0;
 
+    const visibleCases =
+      pagination?.pageSize && totalCases > pagination.pageSize ? pagination.pageSize : totalCases;
+
+    const hasReachedMaxCases =
+      pagination.pageSize &&
+      totalCases >= MAX_DOCS_PER_PAGE &&
+      pagination.pageSize * (pagination.pageIndex + 1) >= MAX_DOCS_PER_PAGE;
+
+    const isDoNotShowAgainSelected = localStorageWarning && localStorageWarning === true;
+
+    const renderMaxLimitWarning = (): React.ReactNode => (
+      <EuiFlexGroup gutterSize="m">
+        <EuiFlexItem grow={false}>
+          <EuiText
+            color="default"
+            size="m"
+            css={css`
+              margin-top: 4px;
+            `}
+          >
+            {i18n.MAX_CASES(totalCases)}
```

Comment:
Shouldn't this be 
```suggestion
            {i18n.MAX_CASES(MAX_DOCS_PER_PAGE)}
```
?

If the users cannot navigate beyond the `limit(MAX_DOCS_...)` and the total > limit we are only going to show a `MAX_DOCS_...` number of cases right?